web-marketplace/src/components/organisms/StoryForm/StoryForm.constants.ts

```diff
@@ -0,0 +1,15 @@
+export const LAND_STORY_LABEL = 'Story of the land';
+export const LAND_STORY_DESCRIPTION = 'Describe the story of this property.';
+export const LAND_STEWARD_STORY_LABEL = 'Story of the land stewards';
+export const LAND_STEWARD_STORY_DESCRIPTION =
+  'Tell the story of who are they, what do they do on the land, what is their background, why are they excited to do this work.';
+export const LAND_STEWARD_STORY_TITLE_LABEL =
+  'Title of the land stewards story';
+export const LAND_STEWARD_STORY_TITLE_DESCRIPTION =
+  'In one sentence, summarize the story above of the land stewards.';
+export const PROJECT_QUOTE_LABEL =
+  'Quote from land steward or other important stakeholder';
+export const PROJECT_QUOTE_DESCRIPTION =
+  'Choose an inspiring quote that helps others understand why this project is important.';
+export const PROJECT_QUOTE_NAME_LABEL = 'Name of person quoted';
+export const PROJECT_QUOTE_JOB_TITLE_LABEL = 'Title of person quoted';
```

Comment:
Looks like these need to be translated, here with `` msg`...` `` and later when being used in `StoryForm.tsx` with `_(...)` unless theres something I'm missing about how the formik `<Form  translate="true">` component works.
I'm having trouble finding this component in the app, is it still used?

Reply:
It's a legacy form not being used anymore, as per this [comment](https://github.com/regen-network/regen-web/blob/dev/web-marketplace/src/components/organisms/StoryForm.tsx#L157-L159).
Should I delete it or update it anyway @blushi ?

Reply:
I think we can just delete it